solutions/security/advanced-entity-analytics/anomaly-detection.md

```diff
@@ -73,6 +83,8 @@ Machine learning jobs look back and analyze two weeks of historical data prior t
 
 ## View detected anomalies [view-anomalies]
 
+From the security solution, you can view the details of detected anomalies within the `Anomalies` table widget shown on the Explore > Hosts, Network, and Users pages, or even narrow to the specific date range of an anomaly from the `Max anomaly score by job` field in the overview of the details pages for hosts and IPs. These interfaces also offer the ability to drag and drop details of the anomaly to Timeline, such as the `Entity` itself, or any of the associated `Influencers`.
```

Comment:
It looks like we call these data tables [elsewhere](https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/2240/solutions/security/explore/hosts-page) in the docs.

What about something like this?

```suggestion
View details of detected anomalies in the **Anomalies** data tables, available on the **Hosts**, **Network**, or **Users** pages. You can access these pages from the navigation menu or with the [global search field](../../explore-analyze/find-and-organize/find-apps-and-objects.md).

You can narrow the time range to a specific anomaly using the `Max anomaly score by job` field in the overview section of the host or IP details. You can also drag and drop anomaly details, like the `Entity` or any associated `Influencers`, into **Timeline**.
```

Reply:
Thanks for the PR! I know you didn't write this part but figured we could clean it up anyway 😄